integration/nodejs/fastify.mdx

```diff
@@ -150,8 +150,43 @@ fastify.get('/admin/hello', async (request, reply) => {
 });
 ```
 
-## PKCE and Client Secrets
+## Authentication Flows
 
-Civic Auth uses [**PKCE** (Proof Key for Code Exchange)](https://oauth.net/2/pkce/), to protect users and clients from unauthorized access to user information. This, alongside domain registration for apps in production environments, mean that you don't need to provide a client secret in your backend.
+Civic Auth supports multiple OAuth 2.0 authentication methods to provide maximum security for different application architectures.
 
-When using the Civic Auth SDK, PKCE is handled entirely by the library.
+<Info>
```

Comment:
see the express case - also let's avoid too much duplication - this information should be in the authentication flows - the note is sufficient here IMO